### ISSUE TITLE
HTML Validation/Required Message doesn't Appear

### ISSUE THREAD

**Issue by javedbaloch4** [snapappointments/bootstrap-select#2342]
Hi,

I am using client side validation in `select` field using `required` validation works but validation message doesn't appear.

**Comment by caseyjhol** [comment]
Bug reports must include a live demo of the problem. Per our [contributing guidelines](https://github.com/snapappointments/bootstrap-select/blob/master/CONTRIBUTING.md), please create a reduced test case (you can use our preconfigured [Plunker](https://developer.snapappointments.com/bootstrap-select/playground/)), and report back with:

- your link
- Bootstrap version
- bootstrap-select version
- specific browser and OS details.

**Comment by Kowal0** [comment]
I made a related issue some time ago https://github.com/snapappointments/bootstrap-select/issues/2304

If you're trying to use HTML validation  [in a way suggested in BS4 docs](https://getbootstrap.com/docs/4.0/components/forms/#validation) then div containing error message has to be validated elements next sibling. When bootstrap-select initiates it creates a container around select element and there's no way of passing error message div into it. 

I solved it by appending my error message into bootstrap-select's container on 'loaded.bs.select' event, but it would be nice if we could just add error messages through options instead.
